Robust.Shared/ContentPack/ResourceManager.cs

```
@@ -177,14 +177,16 @@ public bool TryContentFileRead(string path, [NotNullWhen(true)] out Stream? file
         /// <inheritdoc />
         public bool TryContentFileRead(ResPath? path, [NotNullWhen(true)] out Stream? fileStream)
         {
+            fileStream = null;
+
             if (path == null)
             {
-                throw new ArgumentNullException(nameof(path));
+                return false;
             }
 
             if (!path.Value.IsRooted)
             {
-                throw new ArgumentException($"Path '{path}' must be rooted", nameof(path));
+                return false;
```

This really shouldn't be tolerated. It turns error on invalid input into silent failure.

I don't know why the hell this function's marked as nullable though.

---

Should callers just handle user input then? Because admins will be able to write to it directly.

---

Yes.